Moderation_Matrix.md

```diff
@@ -0,0 +1,58 @@
+# Moderation Matrix
+This matrix correlates with the [Community Rules](https://www.theodinproject.com/community_rules) that are required to be followed in order to participate in the community. Some rules have different levels of severity, so you may see references to them under different numbers of zaps. 
+
+This matrix is not all encompassing and zaps are given at the discretion of the moderator team.
+
+## 0 Zaps (Heads Up)
+- Dog piling/short circuiting a teaching moment
+- Asking about or encouraging the use of Windows
+- "Chat bombing", aka posting many short (ususally 4-5 or fewer words) messages in rapid succession, repeatedly sending parts of thoughts (incomplete, non-standalone sentences or fragments) in new messages. (Use shift + enter and send ONE message where possible or edit your message) 
+- Discussing mental health issues in depth
+
+## 1 Zap (WARNING) 
+Minor violations such as:
+- "Mini-modding"
+  - Please leave moderation action to the moderators, use ModMail to report rule breaking or if you see something you’re unsure about in the community
+- Asking about topics or projects unrelated to The Odin Project 
+-Mild unprofessionalism:
+  - Drug related conversation 
+  - Treating others on the server disrespectfully
+  - Mental health issues discussion 
+  - Relationship advice
+- A second infraction of any 0 Zap issues above
+
+## 2 Zaps (WARNING)
+Medium severity violations such as: 
+- Suggesting piracy 
+- Unsolicited pings/DM/friend request 
+- Political or religious topics 
+- Mild toxicity (directed insults, provoking/baiting other users, passive aggression, naming & shaming)
+- Self promo without permission (ask with Modmail)
+  - Do not share resources you have created solely for personal or monetary gain
+  - Examples: "Fill out my survey" (does not help the goals of Odin), "Join my boot camp/program/server" (Could mislead learners), "Why PHP is superior" (unrelated blog post)
+
+## 5 Zaps
+High severity violations such as: 
+- Posting about any illegal activities, such as facilitating piracy 
+- Publicly arguing about receiving moderation activity after being asked to take it to ModMail/discuss privately 
+- Excessive toxicity (targeted harassment, inciting drama, aggressively arguing purposefully, etc.) 
+
+## 10 Zaps (Immediate Ban)
+Extreme severity violations such as: 
+- Bigotry (racism, homophobia, hate speech etc)
+- Harassment 
+- NSFW Imagery
```

Comment:
Thinking this might be better as "NSFW or otherwise highly offensive imagery in posts, profile pictures, or usernames", or something along those lines?